runtime/libia2/CMakeLists.txt

```diff
@@ -55,15 +55,18 @@ target_link_options(libia2 INTERFACE
 set_property(TARGET libia2 PROPERTY INTERFACE_LINK_DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/padding.ld ${CMAKE_CURRENT_BINARY_DIR}/dynsym.syms)
 
 if (LIBIA2_AARCH64)
+    set(SYSROOT_BIN_DIR ${CMAKE_BINARY_DIR}/external/glibc/sysroot)
+    set(LIBC_PATH ${SYSROOT_BIN_DIR}/lib/libc.so.6)
+    set(LD_LINUX_PATH ${SYSROOT_BIN_DIR}/lib/ld-linux-aarch64.so.1)
     if (LIBIA2_REBUILD_GLIBC)
         ExternalProject_Add(glibc
             SOURCE_DIR ${CMAKE_SOURCE_DIR}/external/glibc
             BINARY_DIR ${CMAKE_BINARY_DIR}/external/glibc
             CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env
                 # ffixed-x18 has to be set here instead of CFLAGS for it to apply to all files
```

Comment:
nit: s/ffixed-x18/compiler flags in the comment

Reply:
Will fix in a follow-up.